TTKDTrainingApp/src/components/VideoWithControls/Controls.js

```diff
@@ -1,80 +1,111 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 
-import {View, TouchableWithoutFeedback, Text, StyleSheet} from 'react-native';
-import ProgressBar from 'react-native-progress/Bar';
+import {Text, StyleSheet, Animated} from 'react-native';
+import Slider from '@react-native-community/slider';
 import Icon from 'react-native-vector-icons/FontAwesome';
 
-import {PROGRESS_BAR_WIDTH, RATES} from './constants';
+import {RATES} from './constants';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+
+const BUTTON_SLOP = {top: 20, left: 20, bottom: 20, right: 20};
 
 const secondsToTime = time => {
-  return `${Math.floor(time / 60)} : ${time % 60 < 10 ? '0' : ''} ${time % 60}`;
+  return `${Math.floor(time / 60)}:${time % 60 < 10 ? '0' : ''}${time % 60}`;
 };
 
 const Controls = props => {
   const {
+    show,
     videoRefs,
     paused,
     handlePlayPausePress,
     duration,
     progress,
     rate,
     setRate,
+    maxWidth,
   } = props;
 
-  const handleProgressPress = e => {
-    const progressBarPosition = e.nativeEvent.locationX;
-    const seekTime = (progressBarPosition / PROGRESS_BAR_WIDTH) * duration;
+  const [animValue] = useState(new Animated.Value(0));
 
+  const onSeek = data => {
+    const seekTime = data * duration;
     Object.values(videoRefs).map(ref => ref.current.seek(seekTime));
   };
 
-  const handleRateTouch = () =>
+  const onFastBackward = () => {
```

Comment:
not sure for this PR or another to try and revert it back to previous fast forwarding cycle

Reply:
I'll do it for this PR. Thanks for catching it!